### ISSUE TITLE
New Training Step by Step Guide 

### ISSUE THREAD

**Issue by wewe0901** [myelintek/web#43]
**Steps** 
- Dataset Selection
- Network Selection
- GPU Selection
- Hyperperamer configuration
    - Hyperperamer Configure options
    - Select Polices
        - Default configuration 
        - Faster training configuration
        - Automatic Configuration Search (Random search)
        -  Automatic Configuration Search (Bayesian optimization)
        - Automatic Configuration Search (Evolutionary optimization) 
    - Hyperparameters (Flow Simon's field dependency in hpo feature)

**Faster training configuration**
- Max Mini-batchsize
- Data Augmentation [TBD]
- Max Learning Rate [TBD]

**Notes**
- When use click create and faster training configuration selected we show Generanting hyperparameter.. After generated than create training job
- Select policy use Multi selct but disable witch not implemented
